rxnetty-common/src/main/java/io/reactivex/netty/client/ClientConnectionToChannelBridge.java

```diff
@@ -138,7 +140,13 @@ private void newConnectionReuseEvent(Channel channel, final ConnectionReuseEvent
             subscriber.onNext(event.getPooledConnection());
             checkEagerSubscriptionIfConfigured(channel);
         } else {
-            event.getPooledConnection().close(false); // If pooled connection not sent to the subscriber, release to the pool.
+            // If pooled connection not sent to the subscriber, release to the pool.
+            event.getPooledConnection().close(false).subscribe(Actions.empty(), new Action1<Throwable>() {
```

Comment:
Can be replaced with lamda style action instead?

Reply:
`
(throwable -> {
			logger.error("Error closing connection.", throwable);
		})```

Reply:
No can do. Source level is set to 1.7, lambda not allowed.